core/foundation/src/TClassEdit.cxx

```diff
@@ -48,6 +48,11 @@ namespace {
             gInterpreterHelper->ShuttingDownSignal();
       }
    };
+
+   void RemoveSpace(std::string_view & s)
+   {
+      while (!s.empty() && s[0] == ' ') s.remove_prefix(1);
```

Comment:
Potential optimization:
```suggestion
      s.remove_prefix(std::min(s.find_first_not_of(" "), s.length()));
```

Reply:
Thank you, but since `find_first_of` most likely goes character-by-character, too, it's unlikely that it saves instructions. For fun, I ran the two through godbolt, and mine had 16 lines of assembly, yours has 22. This doesn't tell us which one is faster, but I will keep the function is it is now.